SU2_CFD/src/solvers/CNEMONSSolver.cpp

```diff
@@ -405,7 +408,10 @@ void CNEMONSSolver::BC_HeatFluxCatalytic_Wall(CGeometry *geometry,
   string Marker_Tag = config->GetMarker_All_TagBound(val_marker);
 
   /*--- Get the specified wall heat flux from config ---*/
-  su2double Wall_HeatFlux = config->GetWall_HeatFlux(Marker_Tag);
+  su2double Wall_HeatFlux = config->GetWall_HeatFlux(Marker_Tag) / config->GetHeat_Flux_Ref();
+  if (config->GetIntegrated_HeatFlux()) {
+    Wall_HeatFlux /= geometry->GetSurfaceArea(config, val_marker);
+  }
```

Comment:
@WallyMaier to let you know: Here in `BC_HeatFluxCatalytic_Wall` the nondim with Heatflux_Red was forgotten. And the NEMO-tribe can (upon merge of this PR) prescribe HeatFlux in [W] instead of [W/m^2] in case someone wants to.

Reply:
Thank you! Good catch!